SourceCgit/SourceCgit.php

```diff
@@ -131,7 +131,8 @@ public function commit( $p_repo, $p_data ) {
 		# i.e. DATA=`echo $INPUT | sed -e 's/ /+/g'`
 		list ( , $t_commit_id, $t_branch) = split(' ', $p_data);
 		list ( , , $t_branch) = split('/', $t_branch);
-		if ($t_branch != $p_repo->info['master_branch'])
+		$master_branches = map( 'trim', explode( ',', $p_repo->info['master_branch']));
```

Comment:
@boecko this triggers *PHP Warning:  Uncaught Error: Call to undefined function map()*, is that supposed to be **array_map** ?